docs/di-govuk-one-login-service-header-design-history.md

```diff
@@ -0,0 +1,216 @@
+
+## Dates the GOV.UK One Login service header has been tested:
+1. 27, 28 February 2023
+2. 17, 18 January 2023
+3. 25, 26 October 2022 (round 2)
+4. 11, 12 October (round 1)
+5. 25, 26 May 2022 (round 2)
+6. 10, 11 May 2022 (round 1)
+
+## How we tested the header
+
+We usability tested various iterations of the header. In testing, we asked participants to create a GOV.UK One Login to use a service, followed by a separate task of signing into a different service with their GOV.UK One Login. In these tasks we asked participants where they thought the links in the header would take them and where they would go to manage their GOV.UK One Login. This meant we could gauge participants' understanding of GOV.UK One Login’s relationship with the services on GOV.UK and which navigation designs were most successful in conveying this.
+
+## Key things we are designing for
+
+We usability tested various iterations of the header. In testing, we asked participants to create a GOV.UK One Login to use a service, followed by a separate task of signing into a different service with their GOV.UK One Login. In these tasks we asked participants where they thought the links in the header would take them and where they would go to manage their GOV.UK One Login. This meant we could gauge participants' understanding of GOV.UK One Login’s relationship with the services on GOV.UK and which navigation designs were most successful in conveying this.
+
+## Key things we are designing for
+
+- helping users understand that the service account sits under an overarching multi-service login
+- showing users they are signed in
+- trying to impact service team's navigation patterns as little as possible
+- allowing users to sign out easily
+- making sure users have a route to their GOV.UK One Login (to manage credentials, etc.) that doesn't get in the way of the task they're trying to complete
+- how do we match user expectation about what happens when they interact with different elements of the navigation design
+- showing users that the service uses GOV.UK One Login
+
+## Things we are still working out
+
+- what happens if a service has more than one level of navigation or types of navigation components
+- other forms of navigation for complex services, we know the header is not ‘one size fits all’
+
+
+## Versions of the GOV.UK One Login service header we have tested
+
+### 14, 15 March 2023  -  (most recent version)
+Desktop:
+![The black GOV.UK header with two links 'Your GOV.UK One Login' and 'Sign out'. Underneath this in a grey header are the service navigation links.](/assets/7-header-desktop.png)
```

Comment:
I'm not sure why these images aren't displaying when Github renders the markdown 🤔 

@danacotoran do you have any thoughts?